package.json

```diff
@@ -66,8 +66,8 @@
     "watch": "npm-watch build:dev",
     "dtslint": "dtslint types",
     "test": "npx jest",
-    "build": "concurrently \"NODE_ENV=production npm run build:webpack\" \"NODE_ENV=production npm run build:babel\"",
-    "build:dev": "concurrently \"NODE_ENV=development npm run build:webpack\" \"NODE_ENV=development npm run build:babel\"",
+    "build": "concurrently \"cross-env NODE_ENV=production yarn build:webpack\" \"cross-env NODE_ENV=production yarn build:babel\"",
```

Comment:
@gregoryfm any specific reason to replace npm here with yarn?

Reply:
Sorry about that. I always use yarn and I replaced in automatic mode, I didn't think about it. :)
I'll return to npm.